app/assets/javascripts/cortex-field_types-core/cells/content_item.js

```diff
@@ -0,0 +1,19 @@
+$('.media-select--featured').click(function (elem) {
+  var id = $(this).data().id;
+  var title = $(this).data().title;
+  var thumb_url = $(this).data().thumb;
+
+  $(".association_content_item_id").val(id);
+
+  $('.content-item-button__selection').remove();
+  $('.content-item-button').append(
+    '<div class="content-item-button__selection">' +
+    '<img src="' + thumb_url + '" height="50px">' +
+    '<div class="content-item-button__selection__text">' +
+    'Selected Media: ' +
+    title +
+    '</div></div>'
+  );
+
+  window.MODALS.featured.close();
```

Comment:
Should this method match the one above (https://github.com/cortex-cms/cortex-plugins-core/pull/32/files#diff-f5165bb3219f906c9714100622beb28eR9) i.e. `window.MODALS.wysiwyg.close();`